### PR TITLE
allow to give away some thrust for yaw control

### PR DESCRIPTION
**Pull request by tumbili**

### CODE REVIEW

**Comment by LorenzMeier** [comment]
@tumbili What about the lower limit case?

File: src/modules/systemlib/mixer/mixer_multirotor.cpp [hunk 1]
```diff
@@ -75,6 +75,10 @@ float constrain(float val, float min, float max)
 	return (val < min) ? min : ((val > max) ? max : val);
 }
 
+float min (float x, float y) {
```
Comment (DrTon) [inline src/modules/systemlib/mixer/mixer_multirotor.cpp]: fminf maybe?

**Comment by tumbili** [comment]
@LorenzMeier I haven't managed to get yaw unstable on very fast descends yet. Maybe some quads with very low hover thrust will have problems. I don't know.

File: src/modules/systemlib/mixer/mixer_multirotor.cpp (hunk 1 above)
Comment (tumbili) [inline src/modules/systemlib/mixer/mixer_multirotor.cpp]: Yeah, good idea!

**Comment by LorenzMeier** [comment]
Ok, so this looks like a safe fix, even if its not the best / final one, so I'm merging it.

@tumbili Please send PRs against the release branch while we're in release mode. I'll cherry-pick this one into both branches.

**Comment by AndreasAntener** [comment]
@LorenzMeier this went to master instead to release_v1.0.0

**Comment by LorenzMeier** [comment]
Its also cherry picked there:
https://github.com/PX4/Firmware/blob/release_v1.0.0/src/modules/systemlib/mixer/mixer_multirotor.cpp#L316

**Comment by AndreasAntener** [comment]
Oh ok, we've flown release yesterday and it still lost yaw, why I was wondering.

**Comment by LorenzMeier** [comment]
In that case we should look at the log. Roman tested it successfully on our setups.

**Comment by AndreasAntener** [comment]
Should we keep #2279 open for further work?

**Comment by AndreasAntener** [comment]
Yes, will share the log

**Pull request merged**